userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -43,15 +47,19 @@ Bill Cycle Day:: The day of the month the invoice is created for an account. Not
 [[catalog]]
 Catalog:: A description of the products, plans, phases, price lists, and rules that are available to customers. A catalog lets Kill Bill know what is being offered for sale and how much to charge for it.
 
+[[ctd]]
+Charged Through Date(CTD) :: The date up to which the entitlement for a subscription has been invoiced. For `IN_ADVANCE` billing mode, this date will often be in the future; for `IN_ARREAR`, this date will often be in the past.
```

Comment:
For me, the word "entitlement" could muddy the waters. I like this statement (from the Migration Guide), which can be followed by the two phrases about IN_ADVANCE and IN_ARREAR: "Whether the billing happens in arrear or in advance, for each subscription, there is a date up to which it was billed for."

Reply:
Agree to remove the `entitlement` piece from the sentence: `The date up to which the subscription has been invoiced for.`